src/index.ts

```diff
@@ -38,8 +40,17 @@ async function handleRequest(request: Request) {
     .filter(val => val.count > 0)
     .map(val => mappers[val.event](val.count))
 
+    let projectName = url.searchParams.get(PROJECT_PARAMETER)
+    if (!projectName) {
+      return new Response(
+        JSON.stringify({
+          error: `Missing "${PROJECT_PARAMETER}" query parameter in request.`,
+        }),
+      )
+    }
+
   const body = JSON.stringify({
-    content: `New changes were made in project!\n${updates.join('\n')}`,
+    content: `New changes were made in ${projectName} project!\n${updates.join('\n')}`,
```

Comment:
I think it would sound more natural if we left it as just as the project name itself
```suggestion
    content: `New changes were made in ${projectName}!\n${updates.join('\n')}`,
```